src/main/java/uk/co/probablyfine/matchers/StreamMatchers.java

```diff
@@ -360,11 +360,11 @@ public void describeTo(Description description) {
     /**
      * The BaseStream must produce exactly the given expected items in order, and no more.
      *
-     * For infinite BaseStreams see {@link #startsWith(T...)} or a primitive stream variant
+     * For infinite BaseStreams see {@link #startsWith(Object...)} or a primitive stream variant
```

Comment:
Is this an automatic fix suggested by doclint? Since it specifies `@type <T> The type of items` I'd expect it to be okay with the generic rather than `Object`

Reply:
Not an automatic fix, but seems to be the correct way of referencing a generically typed vararg parameter. Using `#startsWith(T...)` generates the following error from the javadoc-tool:
```
Error while generating Javadoc:
Exit code: 1 - StreamMatchers.java:363: error: reference not found
      * For infinite BaseStreams see {@link #startsWith(T...)} or a primitive stream variant
                                            ^
StreamMatchers.java:373: warning - Tag @link: can't find startsWith(T...) in uk.co.probablyfine.matchers.StreamMatchers
```